backend/api/Controllers/MissionController.cs

```diff
@@ -384,31 +385,49 @@ public async Task<ActionResult<byte[]>> GetMap([FromRoute] string installationCo
 
         if (area == null)
         {
-            // This is disabled for now as the Area database is not yet populated
+            // This is disabled for now as the Area database is not yet populated. We may want to auto populate it 
```

Comment:
This is outdated after #940 has been merged

Reply:
Yes I have not yet rebased